apps/meteor/client/views/admin/moderation/helpers/ModerationFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import { Grid, GridItem } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React, { useCallback } from 'react';
+
+import FilterByText from '../../../../components/FilterByText';
+import DateRangePicker from './DateRangePicker';
+
+type ModerationFilterProps = {
+	setText: (text: string) => void;
+	setDateRange: (dateRange: { start: string; end: string }) => void;
+};
+
+const ModerationFilter = ({ setText, setDateRange }: ModerationFilterProps) => {
+	const t = useTranslation();
+
+	const handleChange = useCallback(({ text }): void => setText(text), [setText]);
+
+	return (
+		<Grid>
```

Comment:
Avoid using `Grid` its you can pass `DataRangePicker` as children of FilterBytText the have them inline and customize the size if wanted

Reply:
I didn't know about this, but this is interesting. Thank you, done!

Curiosity: Do we avoid Grid whenever possible, or are there some specific cases, where we should use it.

Reply:
Just use `Grid` to render complex structures aiming to have a specific result on mobile

Reply:
Thank you, I understand, I will keep a note of this.